readme.md

```diff
@@ -92,6 +119,7 @@ OD | Order-Dependent flaky tests as defined in [iDFlakies](http://mir.cs.illinoi
 OD-Brit | Order-Dependent Brittle tests as defined in [iFixFlakies](http://mir.cs.illinois.edu/winglam/publications/2019/ShiETAL19iFixFlakies.pdf)
 OD-Vic | Order-Dependent Victim tests as defined in [iFixFlakies](http://mir.cs.illinois.edu/winglam/publications/2019/ShiETAL19iFixFlakies.pdf)
 ID | Implementation-Dependent Tests found by [Nondex](http://mir.cs.illinois.edu/marinov/publications/ShiETAL16NonDex.pdf)
+NIO | Non-Idempotent-Outcome Tests as defined in [ICSE’22 work](). Tests that pass in the first run but fail in the second.
```

Comment:
You can add the following link, which I will fill in later once the camera ready is ready. http://mir.cs.illinois.edu/winglam/publications/2022/WeiETAL22NIO.pdf